### ISSUE TITLE
warnings using Drush 8 with Drupal 9: undefined constant CONFIG_ACTIVE_DIRECTORY

### ISSUE THREAD

**Issue by jameswilson** [drush-ops/drush#4892]
**Describe the bug**

Using Drush 8.4.8 (stable) with Drupal 9.2.7 (stable) on php 7.4.20 I see the following errors in cli running any drush command.

```
[warning] array_keys() expects parameter 1 to be array, null given config.drush.inc:414
[warning] Use of undefined constant CONFIG_ACTIVE_DIRECTORY - assumed
    'CONFIG_ACTIVE_DIRECTORY' (this will throw an Error in a future version of PHP)
    config.drush.inc:415
[warning] Use of undefined constant CONFIG_SYNC_DIRECTORY - assumed 
    'CONFIG_SYNC_DIRECTORY' (this will throw an Error in a future version of PHP) 
    config.drush.inc:423
[warning] Use of undefined constant CONFIG_SYNC_DIRECTORY - assumed 
    'CONFIG_SYNC_DIRECTORY' (this will throw an Error in a future version of PHP)
    config.drush.inc:471
```
**To Reproduce**

- Drush 8.4.8 w/  Drupal 9.2.7.
- Execute any drush command, like `drush st`  `drush cim` `drush cex`, etc.

**Expected behavior**
Command executes normally without warning messages.

**Actual behavior**
Command executes normally, but with aforementioned warning messages. There is no apparent issues executing the command, its just the console output that is worrying.

### System Configuration
| Q               | A
| --------------- | ---
| Drush version?  | 8.4.8
| Drupal version? | 9.2.7
| PHP version     | 7.4.20
| OS?             | Mac, using drush inside Ddev container via `ddev drush`

**Comment by andypost** [comment]
It's already fixed via https://github.com/drush-ops/drush/pull/4839

**Comment by jameswilson** [comment]
This was resolved by searching the codebase for instances of `CONFIG_SYNC_DIRECTORY` (case sensitive).

I found a single instance of the constant `CONFIG_SYNC_DIRECTORY` constant in `scripts/composer/ScriptHandler.php::createRequiredFiles()`. 

This project started with composer-project (https://github.com/drupal-composer/drupal-project/)  and was at some point migrated to `core-recommended` following docs that I cannot find at the moment.  Nothing was mentioned about updating this file.

The solution was to overwrite `scripts/composer/ScriptHandler.php` with the latest Drupal 9 compatible version from here:

https://github.com/drupal-composer/drupal-project/blob/9.x/scripts/composer/ScriptHandler.php

This resolved the error messages from drush on cli.
